src/com/github/alex_moon/porcupi/manager/ManagerServer.java

```diff
@@ -5,10 +5,12 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import com.github.alex_moon.porcupi.messages.Message;
+
 import com.github.alex_moon.porcupi.Config;
 import com.github.alex_moon.porcupi.handlers.Handler;
 
-public class ManagerServer extends Thread {
+public class ManagerServer extends Thread implements Manager {
```

Comment:
For that matter what in the heck is a `Manager`? Basically a `Manager` is anything that pertains to the shell - soo... things that are going to be context-aware? Well, not really - that's the individual threads therein. Is it worth making this distinction?